docs/extending/config_back_office.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Configure default upload locations, pagination limits, date and time, and more settings for the Back Office.
```

Comment:
The tab title is currently taken from the name in mkdocs.yaml, so it's plain "Configuration (Ibexa Documentation)" at the moment, which is not best for search. To keep the short title in TOC, but long in tab title, you can add title metadata:

```suggestion
title: Back Office configuration
description: Configure default upload locations, pagination limits, date and time, and more settings for the Back Office.
```

Note, currently the tab title will be missing the "(Ibexa Documentation)" part, but it's an error in the theme, I'll fix it.